projects/aries-markets/index.js

```diff
@@ -0,0 +1,51 @@
+const sdk = require("@defillama/sdk");
+const { default: BigNumber } = require("bignumber.js");
+const { getResources, } = require("../helper/aptos");
+const { transformBalances } = require("../helper/portedTokens");
+
+let resourcesCache
+
+async function _getResources() {
+  if (!resourcesCache) resourcesCache = getResources("0x9770fa9c725cbd97eb50b2be5f7416efdfd1f1554beb0750d4dae4c64e860da3")
+  return resourcesCache
+}
+const extractCoinAddress = (str) => str.slice(str.indexOf("<") + 1, str.lastIndexOf(">"));
+const reserveContrainerFilter = (i) => i.type.includes("0x9770fa9c725cbd97eb50b2be5f7416efdfd1f1554beb0750d4dae4c64e860da3::reserve::ReserveCoinContainer")
+
+module.exports = {
+  timetravel: false,
+  methodology:
+    "Counts the lamports in each coin container in the Aries contract account.",
+  aptos: {
+    tvl: async () => {
+      const balances = {};
+      const data = await _getResources()
+      const coinContainers = data.filter(reserveContrainerFilter)
+        .map((i) => ({
+          lamports: i.data.underlying_coin.value,
+          tokenAddress: extractCoinAddress(i.type),
+        }));
+
+      coinContainers.forEach(({ lamports, tokenAddress }) => {
+        sdk.util.sumSingleBalance(balances, tokenAddress, lamports);
+      });
+
+      return transformBalances("aptos", balances);
+    },
+    borrowed: async () => {
+      const balances = {};
+      const data = await _getResources()
+      const coinContainers = data.filter(reserveContrainerFilter)
+        .map((i) => ({
+          lamports: BigNumber(i.data.collateralised_lp_coin.value - i.data.underlying_coin.value).toFixed(0),
```

Comment:
This is not correct actually, but it can serve as a proxy for now. We will send in PR to update the borrow too.

`collateralised_lp_coin` is the amount of LP tokens (which needs to convert to the underlying coin using an exchange rate, think of it like Compound's `cToken`) that is used as collateral.

`underlying_coin` is the number of tokens that counts as TVL (the part that hasn't been lent out.).

We store our borrowed amount in an Aptos' `table`, not sure if there is an existing utility to look up `table` entries.

Reply:
@leofisG thanks for reviewing the code, yeah I thought so, unfornately, I am yet to add a helper for looking up table entries, will try to add it soon.